app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
```

Comment:
The ERD Diagram you are using is not the one[ in the requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md)

Here is the actual ERD diagram we have inside of the requirements
[ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)

Reply:
@wickathou 

Yes, It is in the project requirement. The ERD diagram is a representation of the relationships and associations between tables, surely you cannot expect that to contain Rails conventions, can you?
Please implement the required changes, and focus on the logic of what is being explained. 

At Microverse we teach our students a growth mindset. A growth mindset is the belief that one's abilities and skills can be improved through effort and learning. A fixed mindset is the belief that one's abilities and skills are innate and unchangeable. Having a growth mindset can lead to better outcomes in education, work, and personal development. A growth mindset can help you overcome challenges, embrace feedback, and persist in the face of setbacks. A fixed mindset can limit people's potential, make them avoid criticism, and give up easily when faced with difficulties. Therefore, it is important to cultivate a growth mindset and avoid a fixed mindset in order to achieve one's goals and aspirations.

Kindly focus on learning instead of fighting the people who are trying to help learn the best way to implement solutions. Thank you for your co-operation🙏🏾